source/expressions.tex

```diff
@@ -2708,10 +2708,8 @@
 into the \grammarterm{return-type-requirement} is performed.
 
 \item
-The contextually-determined type being constrained
-by the \grammarterm{type-constraint} is \tcode{decltype((E))}.
-The immediately-declared constraint\iref{temp} of \tcode{decltype((E))}
-shall be satisfied.
+The immediately-declared constraint\iref{temp}
+of the \grammarterm{type-constraint} for \tcode{decltype((E))} is introduced.
```

Comment:
Don't we still need to say that the immediately-declared constraint shall be satisfied? As with the other comment, I don't think the "is introduced" here means anything; perhaps replacing "is introduced" with "shall be satisfied" in the revised wording would work?

Reply:
Applied.